src/OrleansTelemetryConsumers.Counters/OrleansPerformanceCounterInstaller.cs

```diff
@@ -35,7 +37,7 @@ public OrleansPerformanceCounterInstaller()
             if (GrainTypeManager.Instance == null)
             {
                 var loader = new SiloAssemblyLoader(new Dictionary<string, SearchOption>());
-                var typeManager = new GrainTypeManager(false, null, loader); // We shouldn't need GrainFactory in this case
+                var typeManager = new GrainTypeManager(false, loader, DefaultPlacementStrategy.GetDefaultForTesting());
```

Comment:
Like in other places, this `.GetDefaultForTesting()` feels a bit of smell that asks for explanation.